package.json

```diff
@@ -100,12 +103,14 @@
     "@types/request": "2.48.5",
     "@types/sha256-file": "1.0.0",
     "@types/source-map-support": "0.5.3",
+    "@types/sql.js": "1.4.2",
```

Comment:
Fixed sql.js problem, please see the solution here: https://github.com/pastelnetwork/pastel-electron-wallet/compare/pastel-sqlite/db...pastel-sqlite/db-fix

Reply:
Thanks for the comment.
I updated my code according to your comment and sql-wasm.js issue is fixed perfectly but if I couldn't install @types/sql.js and type error occurs still.

`
import initSqlJs, { Database } from 'sql.js'
Could not find a declaration file for module 'sql.js'. 'D:/git/pastel-electron-wallet/node_modules/sql.js/dist/sql-wasm.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sql.js` if it exists or add a new declaration (.d.ts) file containing `declare module 'sql.js';`
`